environments.json

```diff
@@ -2422,7 +2422,7 @@
     "family": "JavaScriptCore",
     "short": "SF&nbsp;12.1",
     "release": "2019-03-25",
-    "obsolete": false
+    "obsolete": true
```

Comment:
i'd think we usually consider 2 majors not to be obsolete; meaning, 12.1 isn't obsolete until 14 is out.

Reply:
Two not obsolete versions in this PR are 13 and 13.1

Reply:
sure but those are minors, not majors.

Reply:
Browserslist also shows minor Safari updates in https://browserl.ist/?q=last+2+versions. As far as I know, Safari minor versions are actually could be treated as a majors, because they just include all latest Safari TP changes. Major version number is coupled to macOS major version.

Reply:
Right - which means that if we only show 13.1 and 13.0 as not obsolete, then people on a slightly older OS won’t see any relevant results.

Reply:
usually new versions are available on all of versions of macOS which are supported at the moment of release (2 latest versions of macOS)

Reply:
https://developer.apple.com/documentation/safari_release_notes/safari_13_1_beta_release_notes

Reply:
13.1 is available even on three last OS versions

> Safari 13.1 ships with iOS 13.4 and macOS 10.15.4. It’s also available for macOS 10.14.6 and 10.13.6.

Reply:
Fair enough.